sig-data-scientist-experience/charter.md

```diff
@@ -0,0 +1 @@
+To be created by SIG...
```

Comment:
Do we have chairs defined yet for this sig to start spinning it up?

Reply:
Almost.  We have one more round of review to go through and I think we can go back and update this file.

Reply:
This is going to be handled in a separate PR.  Resolving for now.